src/App.jsx

```diff
@@ -14,57 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const startIndex = goods.findIndex(products => products === 'Jam');
```

Comment:
1. move Jam to const INITIAL_VALUE
2. move this logic to func `getSelectedGoodIndex` higher
  
`  const getSelectedGoodIndex = selectedGood => goods.findIndex(good => good === selectedGood);
`